src/assets/i18n/en.json

```diff
@@ -33,7 +33,6 @@
     "unexpected_error_backend": "An unexpected error occurred, check the logs",
     "select_at_least_one_record": "You must select at least one record in the list",
     "mandatory_field": "Required field",
-    "only_number_field": "Only numbers are allowed",
```

Comment:
WL (Wrong Line)

Reply:
I deleted it as we already have the 'error_number_pattern' key, not sure why you say WL here